templates/microcloud/resources.html

```diff
@@ -53,10 +52,65 @@ <h3 class="p-heading--5 u-no-padding--top u-no-margin--bottom">
               <a href="https://ubuntu.com/engage/introduction-to-micro-clouds">Introduction to MicroClouds</a>
             </h3>
             <p>
-              A micro cloud is a new class of infrastructure for on-demand computing at the edge. In this webinar we go over what MicroClouds are and why they should be a part of your edge strategy, finishing with a demo.
+              MicroCloud is the newest addition to Canonical’s cloud infrastructure portfolio. It is a low-touch, open-source cloud solution purpose-built for scalable clusters and edge deployments for all types of enterprises. In this webinar, we go over what MicroClouds are and why they should be a part of your edge strategy, finishing with a demo.
+            </p>
+          </div>
+
+          <div class="col-3 col-medium-2 p-section--shallow">
+            <div class="u-embedded--image">
+              {{ image(url="https://assets.ubuntu.com/v1/ec34de89-microcloud-lts.png",
+                            alt="",
+                            width="568",
+                            height="315",
+                            hi_def=True,
+                            loading="auto|lazy") | safe
```

Comment:
```suggestion
                            loading="lazy") | safe
```
lazy loading for images, could you do it for all images.

Reply:
@immortalcodes @abhigyanghosh30 above the fold images like these should be auto-loaded. Lazy loading will affect performance and should only be done for images that are not in the vp on load